rules/dropbox_rules/dropbox_linked_team_application_added.py

```diff
@@ -0,0 +1,58 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return all(
+        [
+            deep_get(event, "event_type", "_tag", default="") == "app_link_team",
+            deep_get(event, "event_type", "description", default="") == "Linked app for team",
+        ]
+    )
+
+
+def severity(event):
+    # Anything involving non-team members should be High
+    if event.get("involve_non_team_member", False):
+        return "High"
+    return "Low"
+
+
+def title(event):
+    # This will either be "user" or "admin"; find the intersection and use that for the key
```

Comment:
nice! As a person largely unfamiliar with box logs, I really appreciate the explanation 💯

Reply:
@edyesed -- I recalled seeing additional actor types in the API docs the other day and I'm glad I went back to find those.

I added the remaining types in [`0bc4818` (#687)](https://github.com/panther-labs/panther-analysis/pull/687/commits/0bc4818184c82a1ae6ea3134598b56c5f7a375bf).

The API docs can be found [here](https://www.dropbox.com/developers/documentation/http/teams#team_log-get_events) for reference.